Project2015To2017.Console/Program.cs

```diff
@@ -81,5 +84,80 @@ private static void ConvertProject(Options options)
 				analyzer.Analyze(project);
 			}
 		}
+
+		private static IEnumerable<Project> Convert(this ProjectConverter self, string target, ILogger logger)
```

Comment:
I wouldn't move this logic here. I do agree with you that this is a console concern, but this limits the usability of the nupkg for people extending it. 

While the vast majority of users will go through the CLI, I do like the ability to extend the core project. I can see a lot of small extensions being written to fix company specific things. I wouldn't want everyone forking the project for that.

I think we can leave this in the core library for people wrapping the nupkg and if we want to do something different for the console than we can write our own here.